pyomo/common/fileutils.py

```diff
@@ -336,9 +336,9 @@ def find_library(libname, cwd=True, include_PATH=True, pathlist=None):
         return lib
     # Search 3: use ctypes.util.find_library (which expects 'lib' and
     # extension to be removed from the name)
-    if libname.startswith('lib') and _system() != 'windows':
-        libname = libname[3:]
     libname_base, ext = os.path.splitext(os.path.basename(libname))
+    if libname_base.startswith('lib') and _system() != 'windows':
```

Comment:
Are these supposed to be part of this PR?

Reply:
Unfortunately, yes - to get tests passing again.